vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -343,5 +345,31 @@ public LocalDateTime getMax() {
         return PARSER.apply(
             getElement().getProperty("max")
         );
+
+    /**
+     * Gets the internationalization object previously set for this component.
+     *
+     * Note: updating the object content that is gotten from this method will
+     * not update the lang on the component if not set back using
+     * {@link DateTimePicker#setDatePickerI18n(DatePickerI18n)}
+     *
+     * @return the i18n object. It will be <code>null</code>, If the i18n
+     *         properties weren't set.
+     */
+    public DatePickerI18n getI18n() {
+        return i18n;
+    }
```

Comment:
Before this code there seems to be missing "}" to end the getMax() method which also makes the compilation fail in CI.

Reply:
Yeah, rebasing caused an issue, fixed now.